src/index.js

```diff
@@ -10,13 +10,26 @@
  */
 import { render } from "preact";
 import { html } from "htm/preact";
-import { initAdaptivePaletteGlobals } from "./GlobalData";
+import { initAdaptivePaletteGlobals, adaptivePaletteGlobals} from "./GlobalData";
 import "./index.scss";
 
 // Initialize any globals used elsewhere in the code.
 await initAdaptivePaletteGlobals();
 
+import { PaletteStore } from "./PaletteStore";
 import { Palette } from "./Palette";
-import bmwJson from "./keyboards/bmw_palette.json";
 
-render(html`<${Palette} json=${bmwJson}/>`, document.getElementById("bmwKeyCodes"));
+import paletteFileMap from "./keyboards/palette_file_map.json";
+import firstLayer from "./keyboards/palettes.json";
+import goBackCell from "./keyboards/backup_palette.json";
+
+PaletteStore.paletteFileMap = paletteFileMap;
+adaptivePaletteGlobals.paletteStore.addPalette(firstLayer);
+adaptivePaletteGlobals.paletteStore.addPalette(goBackCell);
+//import outputPalette from "./keyboards/output_area.json";
+//adaptivePaletteGlobals.paletteStore.addPalette(outputPalette);
+
+adaptivePaletteGlobals.navigationStack.currentPalette = firstLayer;
+render(html`<${Palette} json=${goBackCell} />`, document.getElementById("backup_palette"));
+//render(html`<${Palette} json=${outputPalette} />`, document.getElementById("output_palette"));
```

Comment:
This commented line can be removed? Same as other related spots: line 29 in this file and this div container in `index.html`?

Reply:
Right.  All of them have been removed.